tests/moses/main/moses-framework.h

```diff
@@ -222,6 +222,32 @@ void moses_test_combo(vector<string> arguments,
     if (f_it != expected_tr_strs.end())
          unlink(cmd_tmp.second.c_str());
 }
+
+// test the atomese program candidates
+void moses_test_atomese(vector<string> arguments,
```

Comment:
Although I know it is no CPU bottleneck, it is better to get the habit of passing vector as reference
```suggestion
void moses_test_atomese(const vector<string>& arguments,
```